masschallenge/impact-api
# [AC-7148] DisplayDisplay expert curation status to staff

### Changes introduced in [AC-7148](https://masschallenge.atlassian.net/browse/AC-7327):
Update directory Entrepreneur profile pages with accurate labels

### How to test
- Check out this branch on `accelerate, frontend and impact`
- Add the ALGOLIA_API_KEY value to the frontend .env file from [OnePassword](https://my.1password.com/vaults/bsk4at5ckdpjyxqnlmxlee4kbi/allitems/bu2r6oh2h6gzuvfzupe5jolurm)
- Run `make build` to help the new variable reflect.
- Run `make run-server` on accelerate(AC-7148)

- Login as demo-admin(staff)
- Navigate to directory [page](http://localhost:1234/directory) and:
-- Search for `Tom Hughes` and:
--- See that his card has `Mentor (functional specialist) and Mentor (functional specialist)`  expert label
--- You can also change another mentor's `mentor_type and judge_type value` and see the directory card reflects the changes
- Navigate to Tom Hughes [profile](http://localhost:1234/directory/people/20863) and see that the left sidebar has
--- Mentor Type and Judge Type section


- Masquerade as this [finalist](http://localhost:8181/admin/simpleuser/user/13884/masquerade/)
- Navigate to directory [page](http://localhost:1234/directory) and:
-- Search for `Tom Hughes` and:
--- See that his card has `Mentor and Mentor`  expert label
--- You can also change another mentor's `mentor_type and judge_type value` and see the directory card reflects the changes
- Navigate to Tom Hughes [profile](http://localhost:1234/directory/people/20863) and see that the left sidebar has (You may have to update Tom Hughes mentor_type and judge_type on your local db for this)
--- Mentor Type and Judge Type section

### Sibling PR:
On [frontend ](https://github.com/masschallenge/front-end/pull/206)
On [Accelerate](https://github.com/masschallenge/accelerate/pull/2513)
